### PR TITLE
Api

### PR DESCRIPTION
**Pull request by fengv1976**
Made the API folder and files

### CODE REVIEW

File: pages/api/job/[id].js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import dbConnect from '../../../utils/dbConnect'
```
Comment (EllairaT) [inline pages/api/job/[id].js:1]: image
double check the path

File: pages/api/job/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import dbConnect from '../../../utils/dbConnect'
```
Comment (EllairaT) [inline pages/api/job/index.js:1]: same issue as before, double check path

**Review by EllairaT — CHANGES_REQUESTED** [review]
minor issue with importing dbConnect, double check the paths and make sure the number of folders you're moving up is correct (ie ../../../dbConnect vs ../../dbConnect)

File: pages/api/job/index.js (hunk 2 above)
Comment (fengv1976) [reply pages/api/job/index.js:1]: had this error before can fix it

File: pages/api/job/[id].js (hunk 1 above)
Comment (fengv1976) [reply pages/api/job/[id].js:1]: had this error before can fix it

**Comment by fengv1976** [comment]
@EllairaT the last bug is eslint I don't know how to fix

image

File: pages/api/job/[id].js [hunk 3]
```diff
@@ -1,13 +1,13 @@
-import dbConnect from '../../../utils/dbConnect'
+import dbConnect from '../../../lib/dbConnect'
 import Job from '../../../models/job.model'
 
-dbConnect();
+dbConnect()
```
Comment (EllairaT) [inline pages/api/job/[id].js:4]: should be connectToDatabase() now

**Review by EllairaT — COMMENTED** [review]
change dbConnect() to connectToDatabase() and we're good to go 👍

File: pages/api/job/[id].js (hunk 3 above)
Comment (fengv1976) [reply pages/api/job/[id].js]: changes are made

**Comment by fengv1976** [comment]
@EllairaT It says there are conflict files but the two files are the same?

**Review by EllairaT — APPROVED** [review]

**Pull request merged**